app/forms/dependant_form/assets_value_form.rb

```diff
@@ -0,0 +1,42 @@
+module DependantForm
+  class AssetsValueForm
+    include BaseForm
+
+    form_for Dependant
+
+    attr_accessor :has_assets_more_than_threshold, :assets_value
+
+    validates(
+      :has_assets_more_than_threshold,
+      presence: { message: ->(form, _) { form.error_message } },
+      unless: :draft?
```

Comment:
unless: :draft? , I guess you don't need that, right?

Reply:
At the moment we don't need but it will be useful when these pages will go into the provider flow.
It's the same reason why we put the html in the `shared` folder.